composer.json

```diff
@@ -24,7 +24,7 @@
     "php-http/message": "^1.1",
     "php-http/curl-client": "*",
     "php-http/mock-client": "*",
-    "php-http/socket-client": "*",
+    "php-http/socket-client": "2.0.0-beta1",
```

Comment:
This is necessary whilst 2.0.0 is in beta. Once it's released this can be reverted to '*'. 

Keeping it as '*' forces a hard dependency on httplug ^v1.0 which causes this PR to break.